client/src/pages/Home/WeSupportYou/WeSupportYou.js

```diff
@@ -1,5 +1,6 @@
 import React, { useState } from "react";
 import "./WeSupportYou.css";
+import ImgHands from "../../../image/supporting-elderly-woman-2021-09-24-03-07-01-utc.jpg";
```

Comment:
hi Meg, 
please rename file to 
`supporting-elderly-woman.jpg`
we will need to do the same with the rest of the image files

Reply:
Thanks for your quick review!!